CHANGELOG.rst

```diff
@@ -1,6 +1,39 @@
 Change log
 ==========
 
+0.19.0 (2020-08-??)
+-------------------
+
+BREAKING CHANGES
+~~~~~~~~~~~~~~~~
+
+* **migrate to Python3.7**
+* **disable entry points and switch to ``python -m`` calls**
+* **rename ``_universum`` to ``universum``** *(configs need to be fixed)*
+
+New features
+~~~~~~~~~~~~
+
+* **github-handler:** add :doc:`a new Universum mode <args_github_handler>` to serve as a GitHub App
+* **pylint:** allow selecting exact python version
```

Comment:
AFAIR it allows selecting any version.
Previously it only allowed to select python 2 or 3.
I understand what is meant by "exact version", but it is not clear enough for reader.